core/domain/prod_validation_jobs_one_off.py

```diff
@@ -829,7 +829,11 @@ def entity_classes_to_map_over(cls):
 
 
 class UserStatsModelAuditOneOffJob(ProdValidationAuditOneOffJob):
-    """Job that audits and validates UserStatsModel."""
+    """Job that audits and validates UserStatsModel.
+    Note: This job is expected to take a very long time to run since
```

Comment:
Just a note for this and similar docstrings: after the first line, leave a blank line before "Note:". Usually the top-level Python docstring is one line.

Ditto for all others. For the ones below, you can do something like this:

```
"""Cleans up ExplorationRecommendationsModel.

This is done by:
    1. Removing exploration ids from recommendation list of model if exploration ...
    2. ....

NOTE TO DEVELOPERS: Do not delete this job until issue #10809 is fixed.
```

Reply:
Done.